storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -534,9 +534,35 @@ int TianmuHandler::update_row(const uchar *old_data, uchar *new_data) {
  */
 int TianmuHandler::delete_row([[maybe_unused]] const uchar *buf) {
   DBUG_ENTER(__PRETTY_FUNCTION__);
-  DBUG_RETURN(HA_ERR_WRONG_COMMAND);
-}
+  int ret = HA_ERR_INTERNAL_ERROR;
+  auto org_bitmap = dbug_tmp_use_all_columns(table, table->write_set);
```

Comment:
be aware of multi-version issues.

Reply:
This is indeed a problem worth considering. 
Our Tianmu engine currently supports multiple versions when update and delete